package.json

```diff
@@ -18,11 +18,12 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "jscs": "^2.11.0",
+    "lodash": "^3.10.1",
```

Comment:
I intentionally added this to match snyk CLI, but iirc I don't use this in the tests anymore, so will remove in next push to this branch.